src/core/CLucene/index/TermInfosWriter.cpp

```diff
@@ -85,6 +87,16 @@ void STermInfosWriter<T>::add(STerm<T> *term, TermInfo *ti) {
     add(fieldInfos->fieldNumber(term->field()), termTextBuffer.values, length, ti);
 }
 
+template <typename T>
+void STermInfosWriter<T>::add(const TCHAR* field, const T* text, size_t length, TermInfo* ti) {
+    if (termTextBuffer.values == nullptr || termTextBuffer.length < length) {
+        termTextBuffer.resize((int32_t)(length * 1.25));
```

Comment:
Why we set 1.25?

Reply:
Because in the above function void STermInfosWriter<T>::add(STerm<T> *term, TermInfo *ti), it uses 1.25.